__tests__/app.test.js

```diff
@@ -0,0 +1,18 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
```

Comment:
*beforeEach* test is run we should be seeding the database with the testData and then *afterAll* ending the connection to the database. 
This will seed the database before the tests run so the tests will always have fresh data 
Very similar to what we did in mitch's treasures test file